### PR TITLE
Remove submission page for single question #5670

### LINKED ISSUE #5670: Remove submission page for single question

### PR DESCRIPTION
**Pull request by tarun14110**
Fixes #5670

### CODE REVIEW

**Comment by tarun14110** [comment]
@kanghj  the one specifically for submitting to only a single question are used for testing purpose, to test the submission form ?

**Comment by kanghj** [comment]
@tarun14110,the tests (such as `InstructorFeedbackQuestionSubmitPageUiTest`, `InstructorFeedbackQuestionSubmissionEditSaveActionTest`, ...) for the "single question" submission pages (both action test and ui test) can be removed. If there are any test case for that page that tests logic that's also used during in normal submission page, then we can keep those test cases but move them somewhere else (such as one of `InstructorFeedbackSubmitPageUiTest`/`StudentFeedbackSubmitPageUiTest`/`InstructorFeedbackSubmissionEditPageActionTest`/`StudentFeedbackSubmissionEditPageActionTest`,..).

**Comment by tarun14110** [comment]
@kanghj ready for review.

File: src/main/java/teammates/ui/controller/FeedbackSubmissionEditPageData.java [hunk 1]
```diff
@@ -117,11 +117,6 @@ public String getSubmitAction() {
         return submitAction;
     }
     
-    public String getSubmitActionQuestion() {
```
Comment (kanghj) [inline src/main/java/teammates/ui/controller/FeedbackSubmissionEditPageData.java]: Why is this removed? This should be used in `feedbackQuestionSubmissionForm.tag`?

**Comment by kanghj** [comment]
Did you miss out the jsp files such as `studentFeedbackQuestionSubmissionEdit.jsp` and `instructorFeedbackQuestionSubmissionEdit.jsp`

**Comment by tarun14110** [comment]
@kanghj I missed those jsp and tags. sorry, I will get back to you when I am done.

**Comment by tarun14110** [comment]
@kanghj done

File: src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackQuestionSubmitPageUiTest.java [hunk 2]
```diff
@@ -1,130 +0,0 @@
...
-        
-        fq = BackDoor.getFeedbackQuestion("IFQSubmitUiT.CS2104", "Grace Period Session", 1);
-        submitPage = loginToInstructorFeedbackQuestionSubmitPage("IFQSubmitUiT.instr", "Grace Period Session", fq.getId());
-        submitPage.verifyHtmlMainContent("/instructorFeedbackQuestionSubmitPageGracePeriod.html");
-        
-        ______TS("Closed) session");
-        
-        fq = BackDoor.getFeedbackQuestion("IFQSubmitUiT.CS2104", "Closed Session", 1);
-        submitPage = loginToInstructorFeedbackQuestionSubmitPage("IFQSubmitUiT.instr", "Closed Session", fq.getId());
-        submitPage.verifyHtmlMainContent("/instructorFeedbackQuestionSubmitPageClosed.html");
```
Comment (kanghj) [inline src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackQuestionSubmitPageUiTest.java]: @tarun14110, all these html files used for testing should be removed too.

File: src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackQuestionSubmitPageUiTest.java [hunk 3]
```diff
@@ -1,130 +0,0 @@
...
-import org.testng.annotations.Test;
-
-import teammates.common.datatransfer.DataBundle;
-import teammates.common.datatransfer.FeedbackQuestionAttributes;
-import teammates.common.util.AppUrl;
-import teammates.common.util.Const;
-import teammates.test.driver.BackDoor;
-import teammates.test.pageobjects.Browser;
-import teammates.test.pageobjects.BrowserPool;
-import teammates.test.pageobjects.FeedbackQuestionSubmitPage;
```
Comment (kanghj) [inline src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackQuestionSubmitPageUiTest.java]: The page object class `FeedbackQuestionSubmitPage` can be removed too

**Comment by tarun14110** [comment]
@kanghj done

**Comment by kanghj** [comment]
Looks good. :+1:

**Comment by wkurniawan07** [comment]
@tarun14110 can you update this branch to be merged next?
